Serverp/src/ServerThread.java

```diff
@@ -0,0 +1,5 @@
+
+public class ServerThread
```

Comment:
The second is the recommended approach, extending thread is the old, outdated approach.